js/context.js

```diff
@@ -0,0 +1,13 @@
+/*!
+ * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+module.exports = {
+  '@context': {
+    id: '@id',
+    type: '@type',
+    '@protected': true
+
```

Comment:
This doesn't have the `Sha256HmacKey2019` term defined.

Reply:
::facepalms:: one sec..

Reply:
Fixed.